app/index.html

```diff
@@ -12,6 +12,7 @@
         <!-- bower:css -->
         <link rel="stylesheet" href="../bower_components/jquery.layout/dist/layout-default-latest.css" />
         <link rel="stylesheet" href="../bower_components/videojs/dist/video-js/video-js.css" />
+        <link rel="stylesheet" href="../bower_components/jquery-notific8/dist/jquery.notific8.min.css" />
```

Comment:
Do you think using the min version is better here instead of the unminified version? The build process will minify the CSS anyway, but maybe during development it is better to have the unminified version here.
Similarly for the JS include below, an unminified source will be better for debugging (but there are source maps).
What do you think?